sql/plsql/PlSqlParser.g4

```diff
@@ -6301,6 +6302,12 @@ unary_expression
     | quantified_expression
     | standard_function
     | atom
+    | implicit_cursor_expression
+    ;
+
+// https://docs.oracle.com/en/database/oracle/oracle-database/21/lnpls/plsql-optimization-and-tuning.html#GUID-DAF46F06-EF3F-4B1A-A518-5238B80C69FA
+implicit_cursor_expression
+    : SQL PERCENT_BULK_EXCEPTIONS ('.' COUNT | '(' expression ')' '.' ( ERROR_INDEX | ERROR_CODE))
```

Comment:
Please fix whitespacing:

```suggestion
    : SQL PERCENT_BULK_EXCEPTIONS ('.' COUNT | '(' expression ')' '.' (ERROR_INDEX | ERROR_CODE))
```

Reply:
Thanks for your suggestion. I have fixed it.